crates/cheatnet/tests/cheatcodes/deploy.rs

```diff
@@ -155,7 +155,7 @@ fn deploy_contract_on_predefined_address_after_its_usage() {
         matches!(
             output,
             CallResult::Failure(CallFailure::Error { msg, .. })
-            if msg.contains("Requested contract address") && msg.contains("is not deployed")
+            if msg.to_string().as_str().contains("Requested contract address") && msg.to_string().as_str().contains("is not deployed")
```

Comment:
Do we need both `to_string` and `as_str` here? I think contains should exist on String as well.

Reply:
We don't need `as_str` as compiler automatically casts this to `&str`. @Nemezjusz could you remove that?